pkg/odo/cli/env/env_test.go

```diff
@@ -45,8 +45,20 @@ func TestPrintSupportedParameters(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			got := strings.TrimSpace(printSupportedParameters(tt.supportedParameters))
-			if !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("Got %s, want %s", got, tt.want)
+
+			gotStrings := strings.Split(got, "\n")
+			wantStrings := strings.Split(tt.want, "\n")
+
+			gotStringMap := make(map[string]bool)
+			for _, gotString := range gotStrings {
+				gotStringMap[gotString] = true
+			}
+
+			for _, wantString := range wantStrings {
```

Comment:
Did you try to use `DeepEqual` instead? Because, technically, you should also check that there is no extra strings in `gotStrings`, which would be wrong (i.e. there is a parameter in `gotStrings` that is not supposed to be there, which this doesn't check.

Reply:
Thanks :+1:  I missed that. I have updated the logic to use DeepEqual and sorting.